clusterDeployer.py

```diff
@@ -240,6 +245,24 @@ def _validate_external_port(self, lh):
             return extif
         return False
 
+    def _validate_api_port(self, lh):
```

Comment:
clearly this is a method on host, not in clusterDeployer.

Reply:
I think both _validate_api_port() and _validate_external_port() belongs to clusterDeployer: they validate if piece of configuration is valid, using configuration information and the host.